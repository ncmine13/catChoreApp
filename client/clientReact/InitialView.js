var React = require('react');

class InitialView extends React.Component {
	constructor(props) {
		super(props);
		this.state = {
			roommatename: '',
			chorename: '',
			entryInfo: []
		}
	}
	render() {
		return (
			<div>
				<h1>Welcome to your chore list! Log a chore.</h1>
			</div>
		)
	}
}

module.exports = InitialView;