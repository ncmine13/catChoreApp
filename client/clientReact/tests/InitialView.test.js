import React from 'react';
import {shallow} from 'enzyme';
import InitialView from '../InitialView';

test('sample', () => {
	expect(true).toBe(true);
})