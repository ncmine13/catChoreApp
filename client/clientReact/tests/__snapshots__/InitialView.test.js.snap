// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InitialView component should render as expected 1`] = `
<div>
  <h1>
    Welcome to your chore list! Log a chore.
  </h1>
</div>
`;
